core/templates/pages/exploration-editor-page/services/exploration-rights.service.ts

```diff
@@ -131,7 +131,8 @@ angular.module('oppia').factory('ExplorationRightsService', [
             data.rights.community_owned, data.rights.viewable_if_private);
         });
       },
-      saveModeratorChangeToBackend: function(emailBody) {
+      saveModeratorChangeToBackend: function(
+          emailBody, onUnpublishingcallback) {
```

Comment:
Any specific reason for having "Unpublish" keyword in the callback? (Also, change it to `onUnpublishcallback` [camleCase])

Reply:
Once the exploration is unpublished and the user exploration permissions have been updated will the callback be invoked/called

Reply:
DONE

Reply:
The function name is `saveModeratorChangeToBackend` and the callback param name  `onUnpublishingcallback` doesn't make sense to me, can you please find a better name? (I'm not aware of where this function is used so unable to suggest a good name!)

Reply:
Changed to `onExplorationRightsUpdatedcallback`. How does that sound?